openedx/core/lib/block_structure/transformers.py

```diff
@@ -61,7 +62,11 @@ def __iadd__(self, transformers):
                 "The following requested transformers are not registered: {}".format(unregistered_transformers)
             )
 
-        self._transformers.extend(transformers)
+        for transformer in transformers:
```

Comment:
Interesting that you went this path.  I had debated whether to sort the transformers at the time they were added to the collection or do it lazily (as in the original PR).  I initially opted for the latter since I wanted to avoid this `for` loop and `if` statement.  However, looking at the code like this, this works!